core/services/cron/delegate.go

```diff
@@ -28,6 +28,10 @@ func (Delegate) OnJobDeleted(spec job.Job) {}
 
 // ServicesForSpec returns the scheduler to be used for running cron jobs
 func (d *Delegate) ServicesForSpec(spec job.Job) (services []job.Service, err error) {
+	// TODO: we need to fill these out manually, find a better fix
```

Comment:
Can we link a story

Reply:
If we can agree on a fix I'd like to just resolve it in this PR